src/extensions/filtercontrol/bootstrap-table-filtercontrol.js

```diff
@@ -113,12 +114,43 @@
                         break;
                     case 'select':
                         html.push(sprintf('<select class="%s form-control" style="width: 100%; visibility: %s"></select>',
-                            column.field, isVisible))
+                            column.field, isVisible));
                         break;
                 }
             }
 
             that.$header.find(sprintf('.th-inner:contains("%s")', column.title)).next().append(html.join(''));
+            if(column.filterData !== undefined && column.filterData.toLowerCase() !== 'column'){
+                var filterDataType = column.filterData.substring(0, 3);
```

Comment:
if you set data-filter-data="url:http://mikepenz.com/jsfiddle/", and if that link return json like {'key1':'value1', ... 'keyn':'valuen'} then the select filter will be filled with that data.